### PR TITLE
adjust template to use built-in UI controls

### PR DESCRIPTION
**Pull request by liupeirong**
1. Use storage account picker
2. Use dns picker
3. Installs LIS4 and reboots
4. Default to centos 6.6
5. Make all keys securestring

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@liupeirong__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: mysql-replication/mainTemplate.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
         },
         "vmSize": {
             "type": "string",
```
Comment (singhkay) [inline mysql-replication/mainTemplate.json]: @liupeirong We're not setup to consume mainTemplate.json outside of marketplace. This is going to break a bunch of testing/validation scenarios. Can you please rename these back to azuredeploy.json? Thanks!

**Comment by liupeirong** [comment]
Marketplace works with mainTemplate, GitHub "DeployToAzure" button, PowerShell ARM commandlets, and crossplat command line tool all work fine.  Can you explain what's the scenario that doesn't work?  We don't want to keep 2 duplicate copies of the file if possible.

**Comment by singhkay** [comment]
We're still thinking through if we want to allow introduction of maintemplate filenames into the repo or rename all existing azuredeploy to maintemplate. Currently there are a few dependencies on the filename including the acomghbot which is setup to validate azuredeploy.json. and the upcoming readme rendering ACOM. The cost associated with renaming all existing files is pretty high as there are a bunch of official docs and blogs that talk about the azuredeploy.json file. 
https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-workload-template-wordpress/
https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-workload-template-sharepoint/

Of course since this is a community contributed project, I welcome thoughts and feedback :)

**Comment by liupeirong** [comment]
Reverted file name for now.  Please merge.

**Pull request merged**